### PR TITLE
Metadata fixes

### PR DESCRIPTION
**Pull request by alexradzin**
FIR-24449
FIR-24467
FIR-24936

### CODE REVIEW

File: src/main/java/com/firebolt/jdbc/metadata/FireboltDatabaseMetadata.java [hunk 1]
```diff
@@ -465,28 +468,30 @@ public String getSQLKeywords() throws SQLException {
 	@ExcludeFromJacocoGeneratedReport
 	@NotImplemented
```
Comment (ptiurin) [inline src/main/java/com/firebolt/jdbc/metadata/FireboltDatabaseMetadata.java:469]: Now that this function is implemented we can probably remove `@NotImplemented`?

Comment (ptiurin) [reply src/main/java/com/firebolt/jdbc/metadata/FireboltDatabaseMetadata.java:469]: Same in the other methods.

File: src/test/java/com/firebolt/jdbc/metadata/FireboltDatabaseMetadataTest.java [hunk 2]
```diff
@@ -257,6 +257,50 @@ void shouldGetDatabaseMinorVersion() throws SQLException {
...
+
+	@ParameterizedTest
+	@CsvSource(value = {",true", "'',true", "abs,true", "SIN,true", "ThisFunctionDoesNotExist,false"}, delimiter = ',')
+	void getFunctions(String functionNamePattern, boolean filled) throws SQLException {
+		String previousFunction = null;
+		int count = 0;
+		for (ResultSet rs = fireboltDatabaseMetadata.getFunctions(null, null, functionNamePattern); rs.next();) {
+			count++;
+			String functionName = rs.getString("FUNCTION_NAME");
+			System.out.println(functionName);
```
Comment (ptiurin) [inline src/test/java/com/firebolt/jdbc/metadata/FireboltDatabaseMetadataTest.java:288]: We probably don't want to print each time we run this test.

**Review by ptiurin — COMMENTED** [review]
Couple of minor comments.

**Comment by ptiurin** [comment]
Also, please check if the SonarCloud security hotspot is valid or not.

File: src/main/java/com/firebolt/jdbc/metadata/FireboltDatabaseMetadata.java (hunk 1 above)
Comment (alexradzin) [reply src/main/java/com/firebolt/jdbc/metadata/FireboltDatabaseMetadata.java:469]: sure. done

File: src/test/java/com/firebolt/jdbc/metadata/FireboltDatabaseMetadataTest.java (hunk 2 above)
Comment (alexradzin) [reply src/test/java/com/firebolt/jdbc/metadata/FireboltDatabaseMetadataTest.java:288]: definitely.

**Pull request merged**